client/signup/steps/plans/index.jsx

```diff
@@ -94,10 +92,7 @@ export class PlansStep extends Component {
 			return this.props.customerType;
 		}
 
-		const siteGoals = this.props.siteGoals.split( ',' );
-		const customerType =
-			getSiteTypePropertyValue( 'slug', this.props.siteType, 'customerType' ) ||
-			( intersection( siteGoals, [ 'sell', 'promote' ] ).length > 0 ? 'business' : 'personal' );
+		const customerType = getSiteTypePropertyValue( 'slug', this.props.siteType, 'customerType' );
```

Comment:
Does this still need a default?

```suggestion
		const customerType = getSiteTypePropertyValue( 'slug', this.props.siteType, 'customerType' ) || 'personal';
```

Reply:
I think it could be important because it might inform what plan is labelled the "recommended" one.